src/js/dataobjects.js

```diff
@@ -1,4 +1,4 @@
-// import {geography_config} from './geography_providers/geography_sa';
+import {defaults, defaultValues as dv} from './defaults';
```

Comment:
I would probably rename the `defaults` file to `defaultValues` and put the additonal methods either into util or a separate `validation` file.